index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Personal Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Odibee+Sans&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <main>
+      <header>
+        <nav class="container">
+          <a href="#" class="logo"
+            ><img src="./Logo_placeholder.png" alt="Logo"
+          /></a>
+          <a href="#" class="menu"
+            ><img src="./Enabled.png" alt="Menu button"
+          /></a>
+        </nav>
```

Comment:
- [ ] @Enning94 great job implementing the changes, kindly ensure only the hamburger menu is covered by the `nav` tag, the logo is outside the nav tag.